### ISSUE TITLE
Dupuytren Disease impact query

### ISSUE THREAD

**Issue by Dupuytren** [demand-driven-open-data/ddod-intake#57]
**Need**: statistics on the details of Dupuytren contracture care. 

**Value**: Dupuytren disease is a common senior condition, but the health care burden cannot be assessed without diagnosis-tagged data. Current treatments are of limited value. Research funding requires data on Dupuytren disease impact in terms of need for care, care provided and cost of care. Large-scale research is needed to identify Dupuytren biomarkers and move treatment options from damage control to prevention

**Limitations in current data**:  Medicare data is available for 2014, but does not include diagnosis codes. (2014 Dupuytren ICD is 728.6). Dupuytren disease is a chronic progressive disease but Dupuytren disease isn't in the MEPS "selected conditions" list. HCUPnet data relates to inpatient and emergency care, neither of which apply to this disorder, typically treated in an office or outpatient setting and requiring prolonged outpatient hand therapy after treatment. Some procedures such as amputation are used for Dupuytren disease but also are used for other conditions. Meaningful data requires diagnosis-procedure data pairs.

**Desired specifications**: annual data on all Medicare provider claims for Dupuytren disease: ICD-9-CM Diagnosis Code 728.6 (prior to 30 Sept 2015) or  ICD-10-CM Diagnosis Code M72.0 (on or after 1 Oct 2015)

**Comment by dportnoy** [comment]
@Dupuytren Thank you for submitting your use case.  Could you please provide the links to the 2014 datasets you referred to.  What other fields do you need from claim and what's the level of granularity that's acceptable for your purposes?

cc: @DamonDavis

**Comment by Dupuytren** [comment]
Hi, David.

The interactive data set is
https://data.cms.gov/Public-Use-Files/Medicare-Provider-Utilization-and-Payment-Data-Phy/ee7f-sh97
The tab delimited file
https://www.cms.gov/apps/ama/license.asp?file=http://download.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/Medicare-Provider-Charge-Data/Downloads/Medicare_Provider_Util_Payment_PUF_CY2014.zip
is
linked from
https://www.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/Medicare-Provider-Charge-Data/Physician-and-Other-Supplier2014.html

I'd like to have a listing of every claim paid for diagnosis code ICD-9-CM
728.6 or ICD-10-CM M72.0 including all of the fields available in
https://data.cms.gov/Public-Use-Files/Medicare-Provider-Utilization-and-Payment-Data-Phy/ee7f-sh97.
The point of this is to analyze the impact and cost of Dupuytren care by
analyzing the absolute number of claims submitted for each category of
service for the care of Dupuytren disease.

I'd like to have a listing of every claim paid for procedure codes CPT
26910, 26951 and the diagnosis code associated with each claim (ICD-9-CM or
ICD-10-CM), including all of the fields available in
https://data.cms.gov/Public-Use-Files/Medicare-Provider-Utilization-and-Payment-Data-Phy/ee7f-sh97.
The point of this is to find the absolute number and percent breakdown of
finger amputations performed for Dupuytren disease versus other diagnoses.

I want to retrieve this information for the calendar year 2014. If this can
be done, the next step will be to retrieve this information for other years.

Thanks!

Charlie

Charles Eaton MD
Executive Director
Dupuytren Foundation
http://Dupuytrens.org
949-287-3387

**Comment by Dupuytren** [comment]
Clarification: "procedure codes CPT 26910, 26951" should be "procedure
codes CPT 26910, 26951, 26952"

Charlie

Charles Eaton MD
Executive Director
Dupuytren Foundation
http://Dupuytrens.org
949-287-3387

**Comment by dportnoy** [comment]
@Dupuytren / Charlie, Thank you for the clarification.  Although you mention getting a “listing of every claim” for the desired CPTs, it sounds like you don’t need individual claims records and are fine with the data aggregated up to the NPI and CPT level, as with the existing PUF.

The PUF already gives us “Number of Services” counts for CPT codes CPT 26910, 26951, 26952 for all diagnosis.  Now you want to get counts for these CPTs specifically for ICD-9-CM 728.6 or ICD-10-CM M72.0.  The later serves as numerator and former as denominator for your purposes.  Did I get that right?  

In terms of years available, I believe since these include “100% final-action line items” and the PUFs are released for an entire year at a time, there are up to 2 year lag times on the data.  But I’d need to verify this.

**Comment by Dupuytren** [comment]
Hi, David -

Here's the rub: the data is incomplete because "Number of services" claim data in these files is used to truncate the listing. Procedures are _only_ listed if the doctor billed Medicare for performing _more than ten_ of that specific procedure code during the year. The majority - likely all - physicians performing 26910, 26951, or 26952 would have performed less than 10 of those procedures for the diagnosis of Dupuytren disease during any calendar year. The same is probably true for most Dupuytren - related procedures. That's why I'd like to access "all" of the data. There are at least 5000 surgeons in the US performing hand surgery, most of whom do some Dupuytren procedures, but because the data set excludes data for procedures performed 10 or less times in a calendar year, data from only a few hundred surgeons can be retrieved - even from the tab delimited file
https://www.cms.gov/apps/ama/license.asp?file=http://download.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/Medicare-Provider-Charge-Data/Downloads/Medicare_Provider_Util_Payment_PUF_CY2014.zip. Take a look for yourself - in this data set, there are no listings for less than 11 claims for Dupuytren specific procedures such as 26040. 

Thoughts?

**Comment by hilaryzainab** [comment]
@Dupuytren Though this isn't a listing of all the provider specific info, I pulled out the specific codes by state to get a summary list. Maybe you have already done this?

It is clear from the '# of Providers' and the '# of Services column that the aggregate data reports those providers with less than 10 procedures per year. However, I had a similar issue in querying the provider level file you linked to as well as the interactive data. Based on the state summaries I had expected a good deal more provider data that was not available.

I hope this may be useful to you as you move forward, even if it doesn't solve the overall issue immediately. 

[Dupuytren.zip](https://github.com/demand-driven-open-data/ddod-intake/files/304709/Dupuytren.zip)

**Comment by Dupuytren** [comment]
Thanks - This confirms the problem. Unfortunately, this prevents me from gathering the exact information I need: (total number of finger amputations for Dupuytren disease)/(total number of amputations for any diagnosis). I'm open to any suggestions to access Medicare data sets without this data limitation.

**Comment by dportnoy** [comment]
@hilaryzainab, thanks for checking into this.  Could you please elaborate on how you identified there's data for < 10 procedures per year.

> It is clear from the '# of Providers' and the '# of Services column that the aggregate data reports those providers with less than 10 procedures per year. 

@Dupuytren, I believe it's a general rule of CMS to not show only results where the sample size is greater than 10.  This is done to reduce the risk of compromising personal and private information.  So in order to get a count of every procedure, I think CMS would need to aggregate up to a higher level.  Would it be sufficient to get the totals by CPT, without regard to NPI?  Do you need any more granularity?

In the mean time, what could the partial data for just NPIs who had >10 procedures tell you?  Could that be a proxy that gets you a bit closer than what you have now?

**Comment by Dupuytren** [comment]
Hi, David -
The >10 procedure data is a dealbreaker to answer my core question of what
percent of finger amputations are performed for Dupuytren disease vs. other
diagnoses. It would be a fluke for one surgeon to perform more than 3
amputations for Dupuytren disease in a single year,  much less 10 or more.

In the meantime, the available data is a valuable resource for patients
looking for a "Dupuytren specialist" because it does make stats available
for providers with relatively high numbers of Dupuytren procedures. I've
already put this together at http://dupuytrens.org/dupuytren-care-providers/
.

Thanks!

Charlie

Charles Eaton MD
Executive Director
Dupuytren Foundation
http://Dupuytrens.org
949-287-3387

**Comment by hilaryzainab** [comment]
@dportnoy Yes, I based this assumption (I should have said) off of the documentation supplement to the data.
'# of Services = number of services provided, the definition for which can vary by the HCPCS code
'# of providers = number of providers within provider state, HCPCS code and place of service ((F)Facility v. (O) Other)

I noticed that there were always more Services than Providers. Based just on a quick comparison there were more providers listed than would have been possible if only those who had completed the Service of interest more than 10x.

**Comment by dportnoy** [comment]
@hilaryzainab, I see where you got the numbers.  It was the [Medicare State/HCPCS Aggregate table, CY2014, Microsoft Excel (.xlsx)](http://www.cms.gov/apps/ama/license.asp?file=http://download.cms.gov/Research-Statistics-Data-and-Systems/Statistics-Trends-and-Reports/Medicare-Provider-Charge-Data/Downloads/Medicare_State_HCPCS_Aggregate_CY2014.zip).  Thanks!

@Dupuytren, given that "Any aggregated records at the HCPCS code and place of service level which are derived from 10 or fewer beneficiaries are excluded to protect the privacy of Medicare beneficiaries", are you OK using state or national level aggregates?  If so, perhaps we could seek to get aggregates for the ICD codes you need.    While it won't give you a listing of providers, it would give insights into costs and impact.  @hilaryzainab is right that once aggregated, it's clear that providers do very few of these procedures per year.  I'm attaching a spreadsheet I created ([Dupuytren_Medicare_National_HCPCS_Aggregate_CY2014.xlsx](https://github.com/demand-driven-open-data/ddod-intake/files/307105/Dupuytren_Medicare_National_HCPCS_Aggregate_CY2014.xlsx)) that shows the # of relevant unique encounters per provider at national-level aggregates.  Depending on the CPT, the number ranges from 1.2 to 1.6 procedures per year.
